redbot/core/core_commands.py

```diff
@@ -2077,6 +2078,32 @@ async def api(self, ctx: commands.Context, service: str, *, tokens: TokenConvert
         await ctx.bot.set_shared_api_tokens(service, **tokens)
         await ctx.send(_("`{service}` API tokens have been set.").format(service=service))
 
+    @api.command(name="list")
+    async def api_list(self, ctx: commands.Context):
+        """Show all external API services that have been set.
+
+        This command does not expose keys or tokens associated with services,
+        but may expose them if tokens have been incorrectly set as services."""
+
+        services = await ctx.bot.get_shared_api_services()
+        if not services:
+            await ctx.send(_("No API services have been set yet."))
+            return
+
+        sorted_services = sorted(services, key=lambda r: str.lower(r))
+        joined = _("Set API services:\n")
+        for key in sorted_services:
+            joined += "{}\n".format(key)
+        for page in pagify(joined, ["\n"], shorten_by=16):
+            await ctx.send(box(page.lstrip(" "), lang="diff"))
```

Comment:
```suggestion
            joined += "+ {}\n".format(key)
        for page in pagify(joined, ["\n"], shorten_by=16):
            await ctx.send(box(page.lstrip(" "), lang="diff"))
```
So that it applies some syntax highlighting
```diff
Set API services:
+ test
+ test2
```